clamonacc/clamav-clamonacc.service.in

```diff
@@ -11,7 +11,26 @@ After=clamav-daemon.service syslog.target network.target
 Type=simple
 User=root
 ExecStartPre=/bin/bash -c "while [ ! -S /run/clamav/clamd.ctl ]; do sleep 1; done"
-ExecStart=@prefix@/sbin/clamonacc -F --log=/var/log/clamav/clamonacc.log --move=/root/quarantine
+ExecStart=@prefix@/sbin/clamonacc --foreground --log=/var/log/clamav/clamonacc.log
+ExecReload=/bin/kill -SIGHUP $MAINPID
+ExecStop=/bin/kill -SIGTERM $MAINPID
+
+##
+## Security Hardening Options
+##
+ProtectClock=yes
+ProtectKernelTunables=yes
+ProtectKernelModules=yes
+ProtectKernelLogs=yes
+ProtectControlGroups=yes
+NoExecPaths=/
+ExecPaths=@prefix@/sbin/clamonacc /bin/kill
```

Comment:
Can you please verify that `clamonacc` binary has `+x` permission set?

Reply:
This turned out to be due to the lack of execute permissions for `/usr/lib` . Since clamonacc is a dynamically linked ELF, the libraries (`.so` files) need to be executed too. Adding `/usr/lib` to ExecPaths solved the issue for me at least. Will include this in the changes I am making.

Reply:
You should probably add `/usr/local/lib` as well.  Or else use the `@CMAKE_INSTALL_FULL_LIBDIR@` variable so it knows where it should look for libs.

Reply:
That seems to be the better option. I'll add `@CMAKE_INSTALL_FULL_LIBDIR@` to the ExecPath list

Reply:
@micahsnyder If you could, can you help me understand the difference between `@CMAKE_INSTALL_FULL_LIBDIR@` and `@CMAKE_INSTALL_PREFIX@` ?

Reply:
@eternaltyro Sorry I failed to reply to this.  `@CMAKE_INSTALL_FULL_LIBDIR@` would probably be the `@CMAKE_INSTALL_PREFIX@` plus the `lib` directory (or perhaps `lib64` or whatever it's gonna be called).